include/boost/astronomy/coordinate/io.hpp

```diff
@@ -24,7 +24,7 @@ template
 >
 std::ostream& operator<< (std::ostream &out, cartesian_representation< CoordinateType, XQuantity, YQuantity, ZQuantity> const& point)
 {
-    out << "Cartesian Representation ( " << point.get_x() << " , " << point.get_y() << " , " << point.get_x() << " )";
+    out << "Cartesian Representation ( " << point.get_x() << " , " << point.get_y() << " , " << point.get_z() << " )";
```

Comment:
Please create a separate PR for this. It's always good to have unrelated changes submitted differently.

Reply:
Alright... I have removed it from this PR. I will create a new one for it.